ec2/sc-ec2-linux-jumpcloud-notebook-v1.0.0.yaml

```diff
@@ -0,0 +1,500 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Service Catalog: EC2 Reference Architecture.'
```

Comment:
shouldn't this mention something about notebook?

Reply:
None of the existing EC2 templates do this, but sure, I can make that change to all the templates in this PR